library/src/main/scala/scala/scalajs/runtime/RuntimeString.scala

```diff
@@ -398,8 +398,8 @@ private[runtime] object RuntimeString {
       throw new IllegalArgumentException
     } else {
       val offsetCp = codePoint - 0x10000
-      NativeJSString.fromCharCode(
-          (offsetCp >> 10) | 0xd800, (offsetCp & 0x3ff) | 0xdc00)
+      NativeJSString
+        .fromCharCode((offsetCp >> 10) | 0xd800, (offsetCp & 0x3ff) | 0xdc00)
     }
```

Comment:
No, this is an undesirable (but tolerated) change. It should be as before: wrapping the arguments should get higher precedence than a "select chain" if the select chain consists of an only select. This is filed as https://github.com/olafurpg/scalafmt/issues/503